arch/risc-v/src/bl60x/bl_schedulesigaction.c

```diff
@@ -0,0 +1,195 @@
+/****************************************************************************
+ * arch/risc-v/src/bl60x/bl_schedulesigaction.c
```

Comment:
can we reuse the common version:
https://github.com/apache/incubator-nuttx/blob/master/arch/risc-v/src/rv32im/riscv_schedulesigaction.c

Reply:
I'll take a look but I would expect so thanks for pointing it out.

I just got modules today so I wanted to at least get the framework in place to try to bring the OS up.